woocommerce/woocommerce-android
# Add ripple effect on click for see report panel

### Description
See report panel for all analytic cards should show ripple effect on click.

### Testing instructions

1. Open the application compiled in debug mode
2. Click con analytics section in bottom bar menu
3. Click on any see report 
4. Complete panel should show ripple effect

### Images/gif

![ezgif-7-c5b7d97b064d](https://user-images.githubusercontent.com/572601/145401625-35688de9-adb4-4568-a889-3c49842d9f8d.gif)
